core/vm/jump_table.go

```diff
@@ -89,6 +82,15 @@ func NewMetropolisInstructionSet() [256]operation {
 		memorySize:    memoryReturnDataCopy,
 		valid:         true,
 	}
+	instructionSet[REVERT] = operation{
+		execute:       opRevert,
+		gasCost:       gasRevert,
+		validateStack: makeStackFunc(2, 1),
```

Comment:
Ah yes, I changed that since a 0x01 gets into the stack after a REVERT, but the opcode placing that on the stack is CALL*, not REVERT, so it was wrong of me to bump it here. Good catch!